src/App.jsx

```diff
@@ -1,10 +1,88 @@
-function App() {
+import './global.css';
+import { Navbar } from './components/navbar/navbar.jsx';
+import { Banner } from './components/banner/banner.jsx';
+import { FeatureWithIcon } from './components/featureWithIcon/featureWithIcon.jsx';
+import { BigHeader } from './components/bigHeader/bigHeader.jsx';
+import { ProductSection } from './components/productSection/productSection.jsx';
+import { TextBox } from './components/textBox/textBox.jsx';
+import { CompanyLogo } from './components/companyLogo/companyLogo.jsx';
+import { SocialIcon } from './components/socialIcon/socialIcon.jsx';
+import { featureBoxes } from './utilities/featureBoxes.jsx';
+import { productSections } from './utilities/productSections.jsx';
+import { textBoxes } from './utilities/textBoxes.jsx';
+import { bigHeaderContent } from './utilities/bigHeaderContent.jsx';
+import { logos } from './utilities/logos.jsx';
+import { socialIcons } from './utilities/socialIcons.jsx';
 
+function App() {
   return (
     <>
-      <h1>hello</h1>
+      <Navbar />
+      <Banner />
+      <section className="feature-boxes">
+        <div className="container">
+          {featureBoxes.map((featureBox, index) => (
+            <FeatureWithIcon key={index} {...featureBox} />
+          ))}
+        </div>
+      </section>
+      <BigHeader {...bigHeaderContent} />
+      <section>
+        <div className="container">
+          {productSections.map((productSection, index) => (
+            <ProductSection key={index} {...productSection} />
+          ))}
+        </div>
+      </section>
+      <section className="text-box-section">
+        <div className="container">
+          {textBoxes.map((textBox, index) => (
+            <TextBox ey={index} {...textBox} />
```

Comment:
I generally avoid using `...` with props because you can't see what props you're passing there and it makes it harder to read